docs/relnotes.md

```diff
@@ -14,6 +14,9 @@ This is the list of changes to Taipy releases as they were published.
 **`taipy-core`** *<br/>2.1
 
    - New data node named `MongoCollectionDataNode`. It represents the data from a MongoDB collection.
+   - Data node caching mechanism now also considered the date and time of the last modification of a file caused
```

Comment:
"also considered" -> "also considers" or "also takes into account"
(and same remark for 'factor' below)

+ 
"limited to only CSV files, Excel files, JSON files and Pickle files"
->
"limited to CSV, Excel, JSON and pickle files only"